src/OrleansAWSUtils/AWSUtilsSiloHostBuilderExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using Orleans.Hosting;
+using OrleansAWSUtils.Configuration;
+
+namespace Orleans.Runtime.Hosting
+{
+    public static class AwsUtilsSiloHostBuilderExtensions
```

Comment:
I don't see a lot of value in separating the identical methods where 1 targets ISiloHostBuilder and the other IServiceCollection into 2 different types. This just pollutes the Orleans.Hosting namespace unnecessarily. Just merge the 2 types.

Reply:
Hmm I disagree. We have that convention in Orleans core too. And it is clearer that way. One is for IServiceCollection and the other is for ISiloHostBuilder.

Reply:
As a general pattern, we tend to have extension functions grouped by what is being extended, so a separate class for IServiceCollection extensions and ISiloHostBuilder extensions makes sense.  I don't think it's a big deal either way in this case, but neither do I see any reason to change this.